mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1194,6 +1197,8 @@ def _get_args_for_metrics(self, extra_metric, eval_df):
                         eval_fn_args.append(self.evaluator_config.get(column))
                     elif param.default == inspect.Parameter.empty:
                         params_not_found.append(param_name)
+                    else:
+                        eval_fn_args.append(param.default)
```

Comment:
Do we need to do this? Isn't this already done by default?

Reply:
In the case that the eval_fn has parameters (predictions, targets=None, metrics=None) for example - if targets is not provided (but has default value) then the eval_fn_args becomes [predictions, metrics] when we want it to be [predictions, None, metrics] and metrics actually gets passed into targets (with metrics taking its own default value)

Reply:
LMK if this doesn't make sense!